processing/src/main/java/io/druid/query/extraction/LookupExtractor.java

```diff
@@ -106,6 +107,10 @@
    * @return A byte array that can be used to uniquely identify if results of a prior lookup can use the cached values
    */
 
-  @Nullable
   public abstract byte[] getCacheKey();
+
+  public boolean isOneToOne()
+  {
+    return false;
```

Comment:
((as per conversation)) suggest adding docs to indicate when and if this should be removed or changed in the future.